extensions/amp-analytics/amp-analytics.md

```diff
@@ -578,7 +578,7 @@ The `visibilitySpec` is a set of conditions and properties that can be applied t
   - `continuousTimeMin` and `continuousTimeMax`: These properties indicate that a request should be fired when (any part of) an element has been within the viewport for a continuous amount of time that is between the minimum and maximum specified times. The times are expressed in milliseconds. The `continuousTimeMin` is defaulted to 0 when not specified.
   - `totalTimeMin` and `totalTimeMax`: These properties indicate that a request should be fired when (any part of) an element has been within the viewport for a total amount of time that is between the minimum and maximum specified times. The times are expressed in milliseconds. The `totalTimeMin` is defaulted to 0 when not specified.
   - `visiblePercentageMin` and `visiblePercentageMax`: These properties indicate that a request should be fired when the proportion of an element that is visible within the viewport is between the minimum and maximum specified percentages. Percentage values between 0 and 100 are valid. Note that the upper bound (`visiblePercentageMax`) is inclusive. The lower bound (`visiblePercentageMin`) is exclusive, unless both bounds are set to 0 or both are set to 100. If both bounds are set to 0, then the trigger fires when the element is not visible. If both bounds are set to 100, the trigger fires when the element is fully visible. When these properties are defined along with other timing related properties, only the time when these properties are met are counted. The default values for `visiblePercentageMin` and `visiblePercentageMax` are  0 and 100, respectively.
-  - `repeat`: If this property is set to `true`, the trigger fires each time that the `visibilitySpec` conditions are met. In the following example, if the element is scrolled to 51% in view, then 49%, then 51% again, the trigger fires twice. However, if `repeat` was `false`, the trigger fires once. The default value of `repeat` is `false`. `reportWhen` and `repeat` may not both be used in the same visibilitySpec.
+  - `repeat`: If this property is set to `true`, the trigger fires each time that the `visibilitySpec` conditions are met. In the following example, if the element is scrolled to 51% in view, then 49%, then 51% again, the trigger fires twice. However, if `repeat` was `false`, the trigger fires once. The default value of `repeat` is `false`. `reportWhen` and `repeat` may not both be used in the same visibilitySpec. Once the condition has been met, a list of visibility event [variables](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md#visibility) will be reset. This includes `${firstSeenTime}`, `${lastSeenTime}`, `${firstVisibleTime}`, `${lastVisibleTime}`, `${maxContinuousVisibleTime}` and `${totalVisibleTime}`.
```

Comment:
what do you think about those metrics being reset? how different are they from `maxVisiblePercentage` & `minVisiblePercentage`?

Reply:
good point. We could choose not to reset any metric above. However by doing that publishers are losing a lot of metrics that were previously provided. 
We should maybe consider `${xxx}` and `${localXXX}` to clarify the difference between global and local metrics. 
I'll put this PR on hold, until I get back from @jonkeller. Jon any feedback on what the metrics should be when we send multiple visibility requests.  Thank you

Reply:
true. let's be a bit cautious.

Reply:
I would prefer to not reset these variables. The localXXX idea sgtm, though. Thanks!